### DISCUSSION TITLE
Welcome to XGA Discussions!

### DISCUSSION THREAD

**Discussion by DavidT3** [DavidT3/XGA#922] [Announcements]
## 👋 Welcome!
  This Discussions space will be used as a place for making announcements, and more importantly for users of XGA to connect with one another. Hopefully you can:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Make some new connections
  * Start new scientific collaborations

  To get started, you could star this repository and comment below with an introduction, where you're based, and what you're working on!

**Comment by letokotleto** [comment]
hi everyone:) my name is Violetta, and i am currently obtaining an MSc degree in Astronomy in University of Sussex. i used to work on X-ray Isolated Neutron Stars, and my first contribution to XGA was to implement a blackbody fit model. nowadays i am working with dark matter and galaxy cluster, trying to determine the properties that Axion-Like Particles need to have in order to be Dark Matter by observing the influence they will have on the spectrum of a QSO shining through a highly magnetised Galaxy Cluster.

**Comment by gracebrba** [comment]
Hello everyone, my name is Grace and I am a 4th year MPhys Physics with Astrophysics student at the University of Sussex. For my final year project, I am trying to develop a method of categorising galaxy clusters into relaxed or non-relaxed. At the moment, I am using XGA to work out the temperatures within different regions of clusters to try and confirm whether or not they host a cool core by comparing results to previous studies.
If I have time, I will then use XGA to produce gas density profiles of the cluster sample. A peakier profile towards the centre is another good indicator of a cluster hosting a cool core.
